src/plays/airbnb-experiences-clone/components/Card.jsx

```diff
@@ -0,0 +1,39 @@
+import katieZaferesImg from '../assets/images/katie-zaferes.png';
+import weddingImage from '../assets/images/wedding-photography.png';
+import mountainBikeImage from '../assets/images/mountain-bike.png';
+import starImage from '../assets/images/star.png';
+
+export default function Card(props) {
+  let badgeText;
+  if (props.openSpots === 0) {
+    badgeText = 'SOLD OUT';
+  } else if (props.location === 'Online') {
+    badgeText = 'ONLINE';
+  }
+
+  let cardImg;
+  if (props.coverImg === 'katie-zaferes.png') {
+    cardImg = katieZaferesImg;
+  } else if (props.coverImg === 'wedding-photography.png') {
+    cardImg = weddingImage;
+  } else if (props.coverImg === 'mountain-bike.png') {
+    cardImg = mountainBikeImage;
+  }
+
+  return (
+    <div className="card">
+      {badgeText && <div className="card--badge">{badgeText}</div>}
+      <img className="card--image" src={cardImg} />
+      <div className="card--stats">
+        <img className="card--star" src={starImage} />
```

Comment:
add `alt` tag here also

Reply:
@priyankarpal okay will add alt tag for all img elements.